edsdme/blocks/announcements-preview/announcements-preview.js

```diff
@@ -0,0 +1,147 @@
+import { getCaasUrl } from '../../scripts/utils.js';
+import { getConfig } from '../utils/utils.js';
+
+export default async function init(el) {
```

Comment:
Let's move all the functions out of init function I don't see a reason for them to be inside the `init` function